OneBlink.SDK/EmailClient.cs

```diff
@@ -65,27 +50,42 @@ public async static Task<string> SendEmail(string HtmlBody, Attachment[] attachm
                 }
             }
         }
+        private static void AddFileAttachments(FileAttachment[] attachments, BodyBuilder bodyBuilder)
+        {
+            if (attachments != null)
+            {
+                foreach (FileAttachment attachment in attachments)
+                {
+                    using (FileStream fileStream = new FileStream(attachment.filePath, FileMode.Open, FileAccess.Read))
+                    {
+                        bodyBuilder.Attachments.Add(attachment.fileName, fileStream);
+                    }
```

Comment:
Would we be better to go back to a single `Attachment` class with two constructors? One for a filePath and this code would just turn it into a stream in the constructor and another constructor with that just takes a stream. Then devs could use the single `SendEmail()` method with stream attachments and file path attachments. With this implementation, a dev could not send a single email with one stream attachment and one file path attachment.

Reply:
As always, please feel free to tell me I am a crazy person and this implementation is fine :)